src/views/Settings.vue

```diff
@@ -120,6 +115,7 @@ import { DateTime } from 'luxon';
 import TimeZoneModal from '@/views/TimezoneModal.vue';
 import { Actions, hasPermission } from '@/authorization';
 import { translate } from "@hotwax/dxp-components"
+import { DxpAppVersionInfo } from '@hotwax/dxp-components/lib/components';
```

Comment:
After importing the component, register the same in the components. Also import the component from `@hotwax/dxp-components` instead of `@hotwax/dxp-components/lib/components`